src/components/remote-process.ts

```diff
@@ -3,6 +3,10 @@ import { StreamError, streamLogs } from "../api";
 import { ColoredConsole, coloredConsoleStyles } from "../util/console-color";
 import { fireEvent } from "../util/fire-event";
 
+export class ESPHomeBlurSecrets {
```

Comment:
Why is this inside `remote-process` ? This can be in it's own file and doesn't have to be a class

Reply:
I din't think it needed its own file as it is so small - and it needs to be a class as it is used as a static/global variable, I find this a better approach than using the `window` object to set globals...